fastchat/train/train_lora.py

```diff
@@ -195,8 +207,26 @@ def train():
         )
 
     if training_args.local_rank == 0:
-        model.save_pretrained(training_args.output_dir, state_dict=state_dict)
+        # Save full model weights (including lora weights) to lora_weight_path
```

Comment:
Would you be able to elaborate what these lines of code are achieving? If I'm reading this correctly, I think we are saving both the full model weights AND the Lora weights (which is what `model.save_pretrained` achieves). However, I think that for most users, we will only need to save the lora weights and later on we can use `apply_lora` if we wanted to merge these weights to our base weights. I don't think is entirely necessary but correct if I'm wrong :)